packages/xdl/src/ApiV2.ts

```diff
@@ -59,15 +59,20 @@ type QueryParameters = { [key: string]: string | number | boolean | null | undef
 
 type APIV2ClientOptions = {
   sessionSecret?: string;
+  authorizationToken?: string;
 };
 
 export default class ApiV2Client {
   static exponentClient: string = 'xdl';
   sessionSecret: string | null = null;
+  authorizationToken: string | null = null;
 
   static clientForUser(user?: APIV2ClientOptions | null): ApiV2Client {
-    if (user && user.sessionSecret) {
-      return new ApiV2Client({ sessionSecret: user.sessionSecret });
+    if (user && (user.authorizationToken || user.sessionSecret)) {
```

Comment:
I'm not 100% sure why we check on the session secret here, that's already done in the constructor. So that would make these checks obsolete?

Reply:
Removed the property-checks 😄